python/pyiceberg/cli/console.py

```diff
@@ -242,6 +242,29 @@ def rename(ctx: Context, from_identifier: str, to_identifier: str) -> None:
     catalog.rename_table(from_identifier, to_identifier)
     output.text(f"Renamed table from {from_identifier} to {to_identifier}")
 
+@run.command()
+@click.argument("identifier")
+@click.option("--type", required=False)
+@click.option("--verbose", type=click.BOOL)
+@click.pass_context
+@catch_exception()
+def list_refs(ctx: Context, identifier: str, type: str, verbose: bool) -> None:
+    """List all the refs in the provided table"""
+    catalog, output = _catalog_and_output(ctx)
+    table = catalog.load_table(identifier)
+    refs = table.metadata.refs
+    if type:
+        type = type.lower()
+        if type != "branch" and type != "tag":
```

Comment:
Nit: I would consider this more Pythonic:
```suggestion
        if type not in {"branch", "tag"}:
```